iceoryx2/src/service/static_config/message_type_details.rs

```diff
@@ -78,6 +79,7 @@ impl MessageTypeDetails {
         payload_start as *const u8
     }
 
+    /// returns pointer of user_header which is the field follows the header field in a structure.
```

Comment:
I wouldn't state this so explicitly since this is not a contract. I think `returns the pointer to the user header` would be just fine.